packages/jest-cli/src/reporters/CoverageReporter.js

```diff
@@ -103,11 +103,13 @@ class CoverageReporter extends BaseReporter {
       aggregatedResults.coverageMap = map;
     } catch (e) {
       console.error(
-        chalk.red(`
+        chalk.red(
+          `
```

Comment:
You are correct! https://github.com/facebook/jest/commit/e45571cf6571f67dcaf1efe94841c31e84f135ca#diff-cb858d9facdeae9910e3be3db8632971L129